CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # AUX Changelog
 
+## V0.11.8
+
+### Date: TBD
+
+### Changes:
+
+-   Bug Fixes
+    -   Tag values that are objects are displayed as JSON.Stringified text. ie `{ field: "myValue" }`
+        -   Known Issue: Modifying these displayed strings will convert the tag value to a string
+    -   When Moving the camera via `player.MoveTo()`, the pan distance is now set correctly so pan limits are absolute.
```

Comment:
@tomyeti Can you put these notes in the `V0.11.7` section below?

Reply:
Ye